frontend/lib/resolution/maybe-const.cpp

```diff
@@ -169,6 +171,23 @@ AdjustMaybeRefs::Access AdjustMaybeRefs::currentAccess() {
   return access;
 }
 
+void AdjustMaybeRefs::constCheckAssociatedActions(const AstNode* ast, RV& rv) {
```

Comment:
is this serving as a placeholder for future work?

Reply:
Yes. It's not called anywhere yet, but I wanted somewhere to put obvious TODO comments